.github/libs/GithubUtils.js

```diff
@@ -200,13 +200,21 @@ class GithubUtils {
         deployBlockers = [],
         resolvedDeployBlockers = [],
     ) {
-        return this.octokit.pulls.list({
+        return this.octokit.paginate(this.octokit.pulls.list, {
             owner: GITHUB_OWNER,
             repo: EXPENSIFY_CASH_REPO,
             state: 'all',
+            sort: 'created',
+            direction: 'desc',
             per_page: 100,
+        }, ({data}, done) => {
+            const oldestPR = PRList[0];
```

Comment:
I am expecting that PRList is ordered and first element is the oldest. Please tell me If this is incorrect.

Reply:
I think that generally with the GitHub API, the first element is the newest. I have this handy bash alias for testing octokit locally:

1. First, create or get an existing personal access token (PAT) for GitHub.
1. Then, put this script in the root of your Expensify.cash folder:

```javascript
const {Octokit} = require('@octokit/rest');

function setupOctokit() {
    return new Octokit({auth: 'INSERT_YOUR_PAT_HERE'});
}

module.exports = setupOctokit;
```

1. (optional) That file should never be added to version control, so add that file to your global gitignore (usually at `~/.gitignore`) so that you don't accidentally commit your PAT.
1. Add this to your shell profile script:

```bash
alias octokit="NODE_PATH=$PATH_TO_E_CASH node -i -e \"let setupOctokit = require( '$PATH_TO_E_CASH/setupOctokit'); let octokit = setupOctokit(); let _ = require('underscore');\""
```

Then you can just type `octokit` in your terminal and it will open an interactive node session with underscore and an authenticated octokit client for you to test read-only commands with.